package.json

```diff
@@ -24,18 +26,23 @@
     "webpack-bundle-analyzer": "^3.8.0"
   },
   "scripts": {
-    "build": "rollup -c --environment target:all",
-    "build:umd": "rollup -c --environment target:umd",
-    "build:module": "rollup -c --environment target:module",
-    "build:bundle": "rollup -c --environment target:bundle",
-    "build:watch": "rollup -c --environment target:all --watch",
-    "build:prod": "NODE_ENV=production rollup -c --environment target:all",
+    "build": "NODE_ENV=production rollup -c --environment target:all",
```

Comment:
This won't work on Windows. Use `cross-env`? I believe other repos we have used that.